lib/rendition.js

```diff
@@ -82,9 +82,9 @@ async function detectContentType(renditionPath) {
         const fileCmdResult = await getContentTypeInformation(renditionPath);
         localMimeType = fileCmdResult.mime;
         localEncoding = fileCmdResult.encoding;
-    } catch (err) { // fallback to using `file-type lib` (won't set encoding)
-        console.log('`file` command failed', err); // err should be empty
-        console.log('Trying to find mimetype based on rendition file analysis (magic numbers)');
+    } catch (err) { /* eslint-disable-line no-unused-vars */
+        // fallback to using `file-type lib` (won't set encoding)
+        console.log('`file` command failed. Trying to find mimetype based on rendition file analysis (magic numbers)');
```

Comment:
Same comment as before: this log statement is confusing, since this is exactly what `file` is doing as well:

```
Trying to find mimetype based on rendition file analysis (magic numbers)
```

Either just drop it or say "falling back to `file-type` library".

Reply:
ack